src/Compute/Compute/Compute.csproj

```diff
@@ -27,6 +27,7 @@
 
   <PropertyGroup>
     <ModuleAlcEntryAssembly>Microsoft.Azure.Commands.Compute.AlcWrapper</ModuleAlcEntryAssembly>
+    <OutputType>Library</OutputType>
```

Comment:
Is it necessary to make this line of change?

Reply:
Not necessarily, might have been added as part of SDK version change. I removed it in the next revision.